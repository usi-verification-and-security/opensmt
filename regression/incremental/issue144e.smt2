(set-logic QF_LRA)
(declare-fun a () Bool)
(declare-fun b () Bool)
(assert (=> a b))
(check-sat)
(push 1)
(assert (=> a (not b)))
(assert (=> (not a) (not b)))
(assert (=> (not a) b))
(check-sat)
(pop 1)
(check-sat)
(assert (=> a (not b)))
(assert (=> (not a) (not b)))
(assert (=> (not a) b))
(push 1)
(check-sat)
(pop 1)
(check-sat)
