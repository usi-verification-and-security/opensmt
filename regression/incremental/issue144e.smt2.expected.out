sat
unsat
sat
unsat
unsat
